src/Symfony/Bundle/FrameworkBundle/Tests/Controller/AbstractControllerTest.php

```diff
@@ -364,6 +364,44 @@ public function testdenyAccessUnlessGranted()
         $controller->denyAccessUnlessGranted('foo');
     }
 
+    public static function denyAccessUnlessGrantedSetsAttributesAsArrayProvider()
```

Comment:
should be named `provideDenyAccessUnlessGrantedSetsAttributesAsArray`
and be moved after the corresponding test case